postgres_metrics/views.py

```diff
@@ -22,6 +22,7 @@ def metrics_view(request, name):
         request,
         "postgres_metrics/table.html",
         {
+            "title": metric.label,
```

Comment:
This is probably the most important of all of those fixes, making sure the page `<title>` is set correctly.